indigo/docs/guides/howto-hello-indigo-tyrian.md

```diff
@@ -0,0 +1,418 @@
+or replace `HelloIndigo.scala` with
+[this](https://gist.github.com/hobnob/c24f00936e91a7b7e5d644d19e4f1b32)
+
+Although Indigo builds and exports to Electron natively, for this project
+we'll export directly to HTML and use Yarn to run our web server, with
+ParcelJS to copy and package up our HTML, JS, and CSS.
+
+You can use NPM to install Yarn with the command `npm install yarn`, and then
+install ParcelJS with `yarn add parcel --dev`.
+
+### Setting up the Build
+
+Next you'll need to update either your
+`build.sc` (if using mill) to  with this:
+
+```diff
+   def buildGame() = T.command {
+     T {
+       compile()
+       fastOpt()
+-      indigoBuild()()
+     }
+   }
+
+-  def runGame() = T.command {
+-    T {
+-      compile()
+-      fastOpt()
+-      indigoRun()()
+-    }
+-  }
+-
+   val indigoVersion = "0.12.0"
++  val tyrianVersion = "0.3.0"
+
+   def ivyDeps =
+    Agg(
++     ivy"io.indigoengine::tyrian::$tyrianVersion",
++     ivy"io.indigoengine::tyrian-indigo-bridge::$tyrianVersion",
+      ivy"io.indigoengine::indigo-json-circe::$indigoVersion",
+      ivy"io.indigoengine::indigo::$indigoVersion",
+      ivy"io.indigoengine::indigo-extras::$indigoVersion"
+    )
+```
+
+ or `build.sbt` (if using sbt) to this:
+
+ ```diff
+ ```
+
+What we've done here is add Tyrian and the Tyrian Indigo Bridge to our build.
+Tyrian will deal with all of our HTML and the Indigo Bridge will deal with
+communication between Tyrian and Indigo. We've also removed `runGame` as this
+would usually run Electron.
+
+### Setting up the HTML and ParcelJS
+
+Usually Indigo will generate all of our HTML for us and run it through Electron,
+but for this example we're going to generate our own HTML so that we can
+inject CSS.
+
+Firstly we'll create an `index.html` and `app.js` file that looks like
+[this](https://gist.github.com/hobnob/eaa03bfe2da14562e5f2819078f16d63). You'll
+notice we're using the direct JS output from our build here, which may feel odd.
+What will happen when we run ParcelJS through Yarn is that the HTML will be
+copied to a build directory along with the JS, CSS and any dependant static
+files (such as images) that may be needed.
+
+Now generate an empty `css/main.css`.
+
+Next we'll need to update the `package.json` so that ParcelJS will run when yarn
+starts and copies the static files for Indigo to use. To do this add the
+following to the top of `package.json`:
+
+```json
+"scripts": {
+  "start": "parcel index.html --open --no-cache"
+}
+```
+
+We'll need to add to tell ParcelJS about our static files by adding the
+following to `package.json`:
+
+```json
+"staticFiles": {
+  "staticPath": "assets",
+  "staticOutPath": "assets",
+  "watcherGlob": "**"
+}
+```
+
+We need to add a couple of dependencies. Firstly we'll add the
+dependency for static files by running
+`yarn add parcel-reporter-static-files-copy --dev` from the command line; and
+secondly we'll add `snabbdom` for Tyrian by running `yarn add snabbdom`.
```

Comment:
I think it would be better to remove the snabbdom bit here, do all the parcel static install + config, then add snabbdom as a separate final state. It'll be cleaner for the reader and you'll get a chance to add one line explaining what snabbdom is/is for.